sdk/core/System.ClientModel/src/Message/PipelineMessage.cs

```diff
@@ -65,7 +65,7 @@ public CancellationToken CancellationToken
     // the client-provided classifier or compose a chain of classification
     // handlers that preserve the functionality of the client-provided classifier
     // at the end of the chain.
-    public PipelineMessageClassifier MessageClassifier { get; set; }
+    public PipelineMessageClassifier ResponseClassifier { get; set; }
```

Comment:
Can `PipelineMessageClassifier` classify more than responses?  Seems a strange type/property name combination unless we'd be able to use the same message classifier for other purposes on a message.

Reply:
Note today, but you're right the API could feasibly allow this.  We're renaming it to ResponseClassifier so that the Azure.Core HttpMessage type can share the property without adding a separate property for the ClientModel type.